forever hack (#509)
Consensys/doc.zk-evm: docs/build-on-linea/hackathons.md

[**Join the Consensys Developer Chat**](https://t.me/+rI-iPLacQXQ5MDVh) for IRL hackathons and get connected with the MetaMask, Phosphor, and Verax teams!
:::

## Helpful links

- **Get test ETH and get help**
  - Use our [faucet or bridge to get mainnet or testnet ETH](/use-mainnet/fund). Otherwise, [ping us in our Telegram channel](https://t.me/+JMGAPixWzKQyMDY5) or find us IRL at the hackathon to request test ETH directly!
- **Deploy a smart contract**
  - Follow our [step-by-step quickstart on how to deploy and verify a smart contract](/build-on-linea/quickstart) with a variety of tools like Hardhat, Foundry, and more!
- **Get access to Verax private beta**
  - Send your account address in our [Telegram chat](https://t.me/+rI-iPLacQXQ5MDVh) to get access to Verax private beta! Here's a [quick tutorial](https://docs.ver.ax/verax-documentation/developer-guides/tutorials/from-a-schema-to-an-attestation) on how to leverage Verax.
- [**Walkthroughs and use cases**](https://youtube.com/playlist?list=PLJ06SwdM0bLrA-3EGRji4W0QI8fyA8PyW&si=vQsXrtFVUsXhygJ0)

## The Linea Forever Hack

**Create, Iterate, and Innovate!** :frog:

> Every month, Linea will host a mini, online buildathon to celebrate and reward continuous hacking! We will have a 1ETH pool prize distributed to qualified projects. No need to start from scratch - the only requirement is that you're building with Linea, made progress on your project in the current month, and you've posted about your project on social (and included a frog somewhere)!

Reviewer: ```suggestion
Every month, Linea will host a mini, online buildathon to celebrate and reward continuous hacking! We will have a 1 ETH pool prize distributed to qualified projects. No need to start from scratch - the only requirement is that you're building with Linea, made progress on your project in the current month, and you've posted about your project on social (and included a frog somewhere)!
```

This is a great opportunity to win some small prizes alongside your regular building, iteratively improve on past hackathon projects, and get exposure to the Linea team and community!

**Things to note**
- You can submit on behalf of a formal organization (profit-seeking company or DAO), but priority will be given to independent hackers
- You can make multiple submissions, but we will only award prizes to one submission per user
- You can use an old project and refresh it!
- Bonus points for adding a frog somewhere in your submission :wink: :frog:
- Bonus points will be awarded to projects that incorporate the partner tooling found [here](/build-on-linea/tooling)

> **What are examples of acceptable work**

Reviewer: ```suggestion
**Examples of acceptable work**
```

- A step-by-step, published tutorial on how to build a (maybe frog-themed :frog:) dapp on Linea
- A translated guide on how to build on Linea
- A published video explainer on how to build with Linea or Linea concepts
- Improving the UI of a hackathon project you built before
- Flappy bird, but with frogs :frog: and on-chain

> **What is the judging criteria**

Reviewer: ```suggestion
**Judging criteria**
```

- UX - how easy is it for a user to navigate or how easy it is to follow the tutorial
- Utility - how useful is your project
- Technicality - how complex was this project to build
- WOW factor - a catch-all vibe check for projects that are simply fun to interact with or pushing the boundaries of on-chain use cases

> **What counts as a qualified submission**

Reviewer: ```suggestion
**What counts as a qualifying submission?**
```

If you're submitting a dapp or Farcaster frame...
- Open source, public Github repo
- Non-trivial progress made on your project during that month (as measured by your Github commit history)
- Verified smart contract deployed on Linea Sepolia
- Working product demo link
- Short video explainer (less than 5 minutes) of what your project is and what changes you made that month if it is not a new project
- Posted and tagged on Twitter ([@lineabuild](http://x.com/lineabuild)) or Farcaster ([@linea](http://warpcast/linea))

If you're submitting a tutorial...
- Open source, public Github repo
- Published tutorial link
- Non-trivial progress made on your project during that month (as measured by your Github commit history)
- Bonus points for publishing an accompanying video explainer
- Posted and tagged on Twitter ([@lineabuild](http://x.com/lineabuild)) or Farcaster ([@linea](http://warpcast/linea))

Ultimately, a "high-quality" submission is quite subjective, but you can imagine invalid submissions as something like unusable demos, projects that are simply slightly modified clones of other projects, and otherwise scrapped together hacks that took little to no effort to put together.

> **How do I submit**

Reviewer: ```suggestion
**How to submit**
```

> Find the submission deform linked under the current month.

Reviewer: Maybe a stupid question but what's a deform? If it's a platform/service maybe it needs to be capitalised

### April 2024 (1 ETH pool prize + $1000 USD [efrog](https://twitter.com/efrogs_on_linea) pool prize) :frog:

> This month, you can win both the 1ETH pool prize in addition to extra prize money from the [efrog community](https://twitter.com/efrogs_on_linea).

Reviewer: ```suggestion
This month, you can win both the 1 ETH pool prize in addition to extra prize money from the [efrog community](https://twitter.com/efrogs_on_linea).
```

> **Submissions due 11:59PM EST, April 30th, 2024**

Reviewer: ```suggestion
**Submissions due 11:59PM EST, April 30 2024**
```
